refinery/ui/source/js/user-file-browser/services/user-file.js

```diff
@@ -4,15 +4,24 @@
     .module('refineryApp')
     .factory('userFileService', userFileService);
 
-  userFileService.$inject = ['$resource', 'settings'];
+  // TODO: userFileBrowserFiltersCtrl.filters is what I need, but maybe I need a new service?
```

Comment:
Correct. A ctrl should not be a dependency here, but you can set service data from a ctrl.